pkgs/applications/video/mpv/scripts/mpv-youtube-chat.nix

```diff
@@ -0,0 +1,37 @@
+{ lib, stdenvNoCC, fetchFromGitHub, yt-dlp }:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "mpv-youtube-chat";
+  version = "unstable-2023-08-31";
+
+  src = fetchFromGitHub {
+    owner = "BanchouBoo";
+    repo = "mpv-youtube-chat";
+    rev = "90849ac1d03da4d5facda1e03712b6cd57ca364e";
+    sha256 = "sha256-22rTiUX6gYR7iknKLoKdBLhY8PMtXIIBLnckt52Sf2E=";
+  };
+
+  postPatch = ''
+    substituteInPlace main.lua \
+      --replace "opts['yt-dlp-path'] = 'yt-dlp'" \
+      "opts['yt-dlp-path'] = '${lib.getBin yt-dlp}/bin/yt-dlp'"
+  '';
+
+  dontBuild = true;
+
+  installPhase = ''
+    runHook preInstall
+    install -D -t $out/share/mpv/scripts main.lua
+    runHook postInstall
+  '';
+
+  passthru.scriptName = "main.lua";
+
+  meta = with lib; {
+    description = "mpv script to overlay youtube chat on top of a video using yt-dlp";
```

Comment:
Desc should start with capital letter
https://nixos.org/manual/nixpkgs/unstable/#var-meta-description